apps/src/code-studio/progress.js

```diff
@@ -49,26 +56,23 @@ progress.renderCourseProgress = function (scriptData, currentLevelId) {
 
     // Show lesson plan links if teacher
     if (data.isTeacher) {
-      store.dispatch({
-        type: 'SHOW_LESSON_PLAN_LINKS'
-      });
+      store.dispatch(showLessonPlans());
     }
 
     if (data.focusAreaPositions) {
-      store.dispatch({
-        type: 'UPDATE_FOCUS_AREAS',
+      store.dispatch(updateFocusArea({
         changeFocusAreaPath: data.changeFocusAreaPath,
         focusAreaPositions: data.focusAreaPositions
-      });
+      }));
     }
 
     // Merge progress from server (loaded via AJAX)
     if (data.levels) {
-      store.dispatch({
-        type: 'MERGE_PROGRESS',
-        progress: _.mapValues(data.levels, level => level.submitted ? SUBMITTED_RESULT : level.result),
-        peerReviewsPerformed: data.peerReviewsPerformed,
-      });
+      store.dispatch(mergeProgress(
+        _.mapValues(data.levels,
+          level => level.submitted ? SUBMITTED_RESULT : level.result)
+        ), data.peerReviewsPerformed
+      );
```

Comment:
Not entirely sure I'm convinced yet that data.peerReviewsPerformed belongs in this action vs. in its own. I think part of the problem is that mergeProgress isn't really an action - it's what we want to do when we handle the action. Really the action is something like receiveCourseProgress